### PR TITLE
Update animals table

### PR DESCRIPTION
**Pull request by Uthmanbello**
## Highlights
- Added `species` column to `animals` table
- Added more animals to the table
- Updated the `species` column
- Removed negative values from the `weight_kg` column
- Found the `COUNT`, `MIN`, `MAX`, and `AVG` values of different columns 
#### `COMMIT` check:
transaction commit 1
#### `ROLLBACK` check:
transaction rollback
#### Remove negative weights:
transaction negative weight
#### Find `COUNT`, `MIN`, `MAX`, and `AVG`:
transaction avg 1
transaction avg 2

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -3,4 +3,13 @@
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-02-03', 0, true, 10.23);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, true, 8.00);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, true, 11.00);
\ No newline at end of file
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, true, 11.00);
+
+/* Update table's data. */
+
```
Comment (BertrandConxy) [inline data.sql:9]: - [x] Good job so far 👏 Please note that you have forgotten to record data of one animal which is `Charmander` as it is in the project requirements. 
image

Therefore, I would suggest that you record it there too 😃 However I saw that in your screenshots it was there among the results, I don't if you forgot to put it here only but if it is the case, you just need to record it here too 👍

File: queries.sql [hunk 2]
```diff
@@ -7,4 +7,24 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name= 'Pikachu';
...
+BEGIN TRANSACTION;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT my_sp;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT my_sp;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT * from animals;
+
+/* Queries for finding aggregates. */
```
Comment (BertrandConxy) [inline queries.sql:45]: - [ ] Among the transactions that you were supposed to record here, there are missing three of them. 

image

image

image

All these transaction steps should be recorded here too so please add them here 😃

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @Uthmanbello, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Uthmanbello) [reply data.sql:9]: @BertrandConxy, 
So sorry, I forgot to record it in the file. I will do this now.
Thanks

**Review by mikethreels — APPROVED** [review]
Hi Uthman, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Uthmanbello** [comment]
@mikethreels,
Thanks a lot 😃

**Pull request merged**